src/originalBudgetTrackingApp/outputVendors/index.js

```diff
@@ -0,0 +1,10 @@
+import path from 'path';
+
+const files = require.context('.', true, /\.\/.+\/index\.js$/);
```

Comment:
I prefer a less dynamic approach. Especially in apis. We are not going to be adding new vendors every day so the value of dynamic code is low IMO. I will spend some time on the output vendors code